sportsref/nfl/players.py

```diff
@@ -177,6 +177,15 @@ def college(self):
         return college
 
     @sportsref.decorators.memoize
+    def collegeid(self):
```

Comment:
Ohh I see. I'm sorry I should have figured that out, I didn't look closely enough. Sounds good, want to change it to college_player_id() just to be super clear? Usually in comments and stuff when I say "{X} ID" I mean an ID of an X, where X is "boxscore", "player", etc. so I think just to be clear it should be college_player_id() or ncaaf_player_id() maybe?